docs/_includes/toc.md

```diff
@@ -0,0 +1,121 @@
+
+<details>
+  <summary><a href="/fprime/INSTALL.html">Install</a></summary>
+    <ul>
+      <li><a href="/fprime/INSTALL.html">Installing F´</a></li>
+      <li><a href="/fprime/UsersGuide/user/autocomplete.html">Installing F´ Console Autocomplete</a></li>
+    </ul>
+</details>
+
+<details>
+  <summary><a href="/fprime/Tutorials/README.html">Tutorials</a></summary>
+    <ul>
+      <li><a href="https://fprime-community.github.io/fprime-tutorial-hello-world/">HelloWorld</a>: An Introduction to F´</li>
```

Comment:
For each of these tutorials, can we remove the extended description, and simply keep the tutorial name/hyperlink?

Reply:
Yeah this one will also be a quick change that I should have time for this week or next :+1: